### ISSUE TITLE
training r3d on UCF101

### ISSUE THREAD

**Issue by ShallowWill** [yoosan/i3d-tensorflow#4]
Hi, Yoosan, thanks for your code.

Did you train the r3d model on UCF101? If so, what is the accuracy you got?

**Comment by yoosan** [comment]
Hi 
Sorry for the late reply, yes r3d also got around 95% acc on ucf101.

**Comment by ShallowWill** [comment]
Thanks for your reply.
Is it trained from scratch or using pre-trained weights?

**Comment by yoosan** [comment]
yes, pre-trained from kinetics.

**Comment by ShallowWill** [comment]
Thanks, is it good for you to make the pre-trianed r3d model on Kinetics public?
